### DISCUSSION TITLE
Wrong rendering of nested ordered and unordered lists

### DISCUSSION THREAD

**Discussion by GarfieldJiang** [mkdocs/mkdocs#3738] [Q&A]
mkdocs 1.6.0, material 9.5.24
 
```
# Big title

## Secondary title

- Title A
  1. Sub item A1.
  2. Sub item A2.
  3. Sub item A3.
 
- Title B
  1. Sub item A1.
  2. Sub item A2.
```

This will be rendered as the following:
![image](https://github.com/mkdocs/mkdocs/assets/1414571/b87108d8-eaef-4795-8d17-6f97e56174e0)

**Comment by pawamoy** [comment]
Try indenting once more:

```md
- Title A
    1. Sub item A1.
    2. Sub item A2.
    3. Sub item A3.
 
- Title B
    1. Sub item A1.
    2. Sub item A2.
```

**Reply by GarfieldJiang** [reply]
Is this the 'standard' way to use markdown?

**Reply by lampensau** [reply]
I personally don't know the exact standard, but that's how it is implemented in python-markdown (ref: https://github.com/Python-Markdown/markdown/issues/3 or https://python-markdown.github.io/#differences), the markdown library used by MkDocs.
